src/component/parser/json/converter/EventDefinitionConverter.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2020 Bonitasoft S.A.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { AbstractConverter, ensureIsArray } from './AbstractConverter';
+import { TDefinitions } from '../../xml/bpmn-json-model/BPMN20';
+import { ShapeBpmnEventKind } from '../../../../model/bpmn/shape/ShapeBpmnEventKind';
+import { TEventDefinition } from '../../xml/bpmn-json-model/baseElement/rootElement/eventDefinition';
+
+export const bpmnEventKinds = Object.values(ShapeBpmnEventKind).filter(kind => {
+  return kind != ShapeBpmnEventKind.NONE;
+});
+
+const eventDefinitionsOfDefinitions: Map<string, ShapeBpmnEventKind> = new Map();
```

Comment:
:eyes: you use a Map whereas we use array everywhere else. This is a 1st step to migrate the others later?

Reply:
I didn't paid attention, and it seems to me that we said that we had to replace the different arrays with maps.
I can do it tomorrow 😉

Reply:
no problem :smile: